app/screens/transaction-detail-screen/transaction-detail-screen.tsx

```diff
@@ -256,6 +207,12 @@ export const TransactionDetailScreen: React.FC<Props> = ({ route }) => {
       </View>
 
       <View style={styles.transactionDetailView}>
+        {settlementVia.__typename === "SettlementViaOnChain" &&
+          settlementVia.transactionHash === null && (
```

Comment:
to help clarify what we're doing here, this check `settlementVia.__typename === "SettlementViaOnChain" &&
          settlementVia.transactionHash === null` should not be in the "view" but before we do the rendering with a proper variable name. like 
          
`const transactionHasBroadcasted = settlementVia.__typename !== "SettlementViaOnChain" &&
          settlementVia.transactionHash !== null`

Reply:
I agree, addressed in https://github.com/GaloyMoney/galoy-mobile/pull/2364/commits/c1bdf32802f534cb47f3f10b20eaafaa852412cd